src/pages/profile.tsx

```diff
@@ -79,18 +83,19 @@ const UserProfile: NextPageWithLayout = () => {
             用户姓名不能为空
           </Alert>
         )}
-        {!show && <Button
-          onClick={() => setShow(!show)}
-          fontSize='sm' variant='brand' fontWeight='500' mb='24px'>
-          修改个人信息
-        </Button>}
-
-        {show && <Button
+        {loaded && <Button
           backgroundColor='orange'
           onClick={handleSubmit}
           fontSize='sm' variant='brand' fontWeight='500' mb='24px'>
```

Comment:
Remove all custom styling including `backgroundColor`, `fontSize`, `fontWeight`.

Also can you help change `mb` to `marginBottom` -- short acronyms are hard to read and thus maintain.

Reply:
Done